### PR TITLE
Remove some outdated python requirements

### PR DESCRIPTION
**Pull request by rosskarchner**
This is an attempt to identify and remove some unnecessary Python dependencies. Please suggest any candidates, or chime in if you think this is overzealous!

## Removals

- django-dynamicresponse: this is alternative to Django REST framework that we've used in the past, but seem to no longer need
- django-simple-feedback-collector: We don't use this anymore at all.
- feedparser: I'm *pretty sure* we don't use this anywhere
- icalendar and vobject: left over from the days where where were parsing and generating ical files
- suds: I don't even remember us needing to interact with SOAP api's.
- TinyMCE: this required a small code change, JobCategory.blurb was a TinyMCE field, but now it uses the standard Wagtail RichTextField.

## Changes

-

## Testing

1.

## Notes

-

## Todos

-

## Checklist

* [ ] PR has an informative and human-readable title
* [ ] Changes are limited to a single goal (no scope creep)
* [ ] Code can be automatically merged (no conflicts)
* [ ] Code follows the standards laid out in the [development playbook](https://github.com/cfpb/development)
* [ ] Passes all existing automated tests
* [ ] Any *change* in functionality is tested
* [ ] New functions are documented (with a description, list of inputs, and expected output)
* [ ] Placeholder code is flagged
* [ ] Visually tested in supported browsers and devices
* [ ] Project documentation has been updated
* [ ] Reviewers requested with the [Reviewer tool](https://help.github.com/articles/about-pull-request-reviews/) :arrow_right:

### CODE REVIEW

File: cfgov/jobmanager/migrations/0013_auto_20171206_1747.py [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+    dependencies = [
+        ('jobmanager', '0012_jobs_have_one_region'),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name='jobcategory',
+            name='blurb',
+            field=wagtail.wagtailcore.fields.RichTextField(null=True, blank=True),
```
[the remark this answers is not in the document]
Comment (rosskarchner) [reply cfgov/jobmanager/migrations/0013_auto_20171206_1747.py]: It didn't under my testing-- I think both editors just store HTML in the blob.

File: cfgov/jobmanager/migrations/0013_auto_20171206_1747.py [hunk 2]
```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8 -*-
```
[the remark this answers is not in the document]
Comment (rosskarchner) [reply cfgov/jobmanager/migrations/0013_auto_20171206_1747.py]: Wil do

[the remark this answers is not in the document]
Comment (rosskarchner) [reply cfgov/jobmanager/migrations/0013_auto_20171206_1747.py]: Fixed in the latest PR

**Comment by chosak** [comment]
@rosskarchner I brought this PR up to date, motivated in part by some requirement inconsistencies identified by @acrewdson. I modified the jobmanager migration so that it depends on the latest one.

I double-checked that the TinyMCE -> RichText field change seems to work seamlessly as you suggested above:

image

**Comment by rosskarchner** [comment]
@chosak thanks for the assist!

**Pull request merged**